package.json

```diff
@@ -60,6 +60,7 @@
     "react-dom": "^16.12.0",
     "react-dropdown": "^1.7.0",
     "react-scroll": "^1.7.16",
+    "react-select": "3.1.0",
```

Comment:
Using this library so we can style the dropdown menu without having to directly worry about functionality, accessibility, etc. Looks to be quite lightweight: https://bundlephobia.com/result?p=react-select@3.1.0